hpcbench/benchmark/osu.py

```diff
@@ -9,8 +9,7 @@
 
 from hpcbench.api import Benchmark, Metric, Metrics, MetricsExtractor
 from hpcbench.toolbox.process import find_executable
-
-# import hpcbench.toolbox.collections_ext
+import logging
```

Comment:
Please put `import logging` in proper location
See https://www.python.org/dev/peps/pep-0008/#id23